### PR TITLE
Fix compilation error and add .gitignore

### PR DESCRIPTION
**Pull request by ghost**
Please review these changes:
There was a compilation error on the trip file, and I have fixed it.
I also added .gitignore for this project.

### CODE REVIEW

**Review by Nickolas-Duong — APPROVED** [review]
sounds good

**Review by Aliuddink — APPROVED** [review]
looks good

File: src/trip.c [hunk 1]
```diff
@@ -1,94 +1,103 @@
...
-        }
-    }
-
-    //if certain options used, add to allowed expenses
-    if(taxi == true)
+    switch (option)
     {
-        addAllowedExpense(day, 10);
+    case 1:
+      rental = get_Car_Rentals();
```
Comment (hoangtuhuynh) [inline src/trip.c:33]: thats good!

**Review by hoangtuhuynh — APPROVED** [review]

**Comment by aj-cpp** [comment]
I will merge it now.

**Pull request merged**